priv/repo/seeds.exs

```diff
@@ -57,3 +57,14 @@ CoursePlanner.Repo.delete_all CoursePlanner.User
       visible: true
     })
 |> CoursePlanner.Repo.insert!
+
+%CoursePlanner.SystemVariable{}
+|> CoursePlanner.SystemVariable.changeset(
+    %{
+      key: "ATTENDANCE_DESCRIPTORS",
```

Comment:
I though descriptor suits better, but I am fine with both. but double check the meaning and see if you still "descriptions" more relevant.